netbox_slm/forms/software_license.py

```diff
@@ -1,8 +1,11 @@
 from django.urls import reverse_lazy
+from django import forms
```

Comment:
Can the `DateField` be imported directly from forms, instead of the whole package?